### PR TITLE
Test Fix LRQA-68207

### PR DESCRIPTION
**Pull request by MoryAlvesBR**

### CODE REVIEW

**Comment by MoryAlvesBR** [comment]
ci:test:sf

**Comment by MoryAlvesBR** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsDateField.testcase [hunk 1]
```diff
@@ -495,7 +495,7 @@ definition {
 
 		Form.viewFieldReference(
 			fieldName = "Date",
-			fieldValue = "Date1234");
+			key_fieldReference = "Date1234");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsDateField.testcase:498]: @MoryAlvesBR, please rename to `fieldReference`. On macro `viewFieldReference`, please make something like:
```
key_fieldReference = "${fieldReference}";
```
Also, please update all calls to this macro to reflect this change. At this moment, it's being used on 2 places only. :wink: 

Thanks in advance

**Comment by MoryAlvesBR** [comment]
@natocesarrego I will need to be deleted these lines in ReferenceCanBeEdited#FormsDateField.testcase. Because these lines are repeatable in the macro

File: portal-web/test/functional/com/liferay/portalweb/macros/Form.macro [hunk 2]
```diff
@@ -856,7 +856,7 @@ definition {
 
 		AssertTextEquals(
 			locator1 = "Form#FIELD_REFERENCE",
-			value1 = "${key_fieldReference}");
+			value1 = "${fieldReference}");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/Form.macro:859]: @MoryAlvesBR, please see my [previous comment](https://github.com/liferay-forms/liferay-portal/pull/700#discussion_r661800317) carefully. You should do the same as it's being done below:
```
key_fieldName = "${fieldName}";
```

**Comment by MoryAlvesBR** [comment]
@natocesarrego Sorry for my misinterpretation of your first comment, I believe I was able to code your requests.

**Comment by MoryAlvesBR** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
@MoryAlvesBR, can you please rebase your code against to the latest branch master of Brian Chan's repo to try to address the source format issue?

Thanks.

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
@MoryAlvesBR, can you please rebase your code against to the latest branch master of Brian Chan's repo to try to solve the source format issue?

Thanks in advance.

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
@MoryAlvesBR, can you please move the prefix of the Jira ticket to the beginning of the commit message?

I mean: `LRQA-68207 Test fix`. This is the reason of the SF issue.

Thanks

**Comment by natocesarrego** [comment]
Resent here: https://github.com/liferay-forms/liferay-portal/pull/756

**Pull request closed without merging**